website/src/css/custom.css

```diff
@@ -583,6 +583,29 @@ nav.pagination-nav a.pagination-nav__link:hover .pagination-nav__link--label {
   flex: none;
 }
 
+@media only screen and (max-width: 1190px) {
+  .navbar .navbar__inner a.navbar__link {
+    display: none;
+  }
+
+  .navbar__toggle {
+    display: inherit;
+  }
+  .navbar-sidebar {
+    display: block;
+  }
+}
+
+@media only screen and (max-width: 1000px) {
+  .sidebar_src-theme-DocSidebar- div.menu {
+    padding-top: 0;
+  }
+
+  .docSidebarContainer_src-theme-DocPage- {
+    border-right: none !important;
+  }
+}
+
```

Comment:
can we give this a kick in the tire with a production build? Just cause I remember having problems with classnames not being the same in prod vs dev in docusaurus...